src/explore-education-statistics-common/src/components/form/__tests__/FormRadioSearchGroup.test.tsx

```diff
@@ -133,14 +131,61 @@ describe('FormRadioSearchGroup', () => {
 
     const searchInput = screen.getByLabelText('Search options');
 
-    await userEvent.type(searchInput, 'Not there');
+    await user.type(searchInput, 'Not there');
 
     expect(await screen.findByText('No results found')).toBeInTheDocument();
 
-    const radios = screen.queryAllByLabelText(
-      /Test radio/,
-    ) as HTMLInputElement[];
-
+    const radios = screen.queryAllByRole('radio');
     expect(radios).toHaveLength(0);
   });
+
+  test('`alwaysShow` values are shown when they do no match the search term', async () => {
```

Comment:
Could have another test case checking that `alwaysShow` values are present when there is no search term as well